File: src/Fast.NET.Core/AdminFactory/EnumFactory/AdminTypeEnum.cs
namespace Fast.NET.Core.AdminFactory.EnumFactory;

/// <summary>
/// 账号类型
/// </summary>
public enum AdminTypeEnum
{
    /// <summary>
    /// 超级管理员
    /// </summary>
    [Description("超级管理员")]
    SuperAdmin = 1,

    /// <summary>
    /// 管理员
    /// </summary>
    [Description("管理员")]
    Admin = 2,

    /// <summary>
    /// 租户管理员
    /// </summary>
    [Description("租户管理员")]
    TenantAdmin = 3,

    /// <summary>
    /// 普通账号
    /// </summary>
    [Description("普通账号")]
    None = 4
}

File: frontend-template/src/api/modules/admin-type-enum.ts
/**
 * Fast.Admin.Core.Enum.System.AdminTypeEnum 账号类型枚举
 * @export
 * @enum {string}
 */
export enum AdminTypeEnum {
    /**
     * 系统默认账号
     * @memberof AdminTypeEnum
     */
    Default = 0,
    /**
     * 超级管理员
     * @memberof AdminTypeEnum
     */
    SuperAdmin = 1,
    /**
     * 系统管理员
     * @memberof AdminTypeEnum
     */
    SystemAdmin = 2,
    /**
     * 租户管理员
     * @memberof AdminTypeEnum
     */
    TenantAdmin = 3,
    /**
     * 普通账号
     * @memberof AdminTypeEnum
     */
    None = 4,
}
